src/dbspro/cli/run.py

```diff
@@ -23,7 +23,7 @@ def main(args):
     # Check if directory exists. If not make one.
     if not os.path.isdir(args.directory):
         os.mkdir(args.directory)
-        logging.info(f'Output directory {args.directory} created.')
+        logging.info(f"Output directory {args.directory} created.")
```

Comment:
The directory has already have been created by `dbspro init` thus these lines can be removes.

Reply:
Woops, forgot to remove this.